### PR TITLE
Dudes

### PR DESCRIPTION
**Pull request by dm-mykhalenych**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
```
Comment (georgelviv) [inline index.html]: indentation!)

Comment (georgelviv) [reply index.html]: On Windows Shift + Alt + F
https://code.visualstudio.com/

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="css/style.css">
+<title></title>
+</head>
+<body>
+  <div class="card text-center">
+  <div class="card-body">
+   <h2 class="card-title">Welcome to todo list app</h2>
```
Comment (georgelviv) [inline index.html]: add input

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+h2 {
+    text-align:center;
```
Comment (georgelviv) [inline css/style.css]: style.css leave it empty

**Review by georgelviv — COMMENTED** [review]
Please don't remove License and Readme files

File: index.html (hunk 1 above)
Comment (dm-mykhalenych) [reply index.html]: I see, i correct my mistake

**Pull request merged**